lnwire/short_channel_id.go

```diff
@@ -34,6 +34,20 @@ func NewShortChanIDFromInt(chanID uint64) ShortChannelID {
 	}
 }
 
+// NewFakeShortChanIDFromInt generates a new short channel that its block
+// height is between 2^17 = 131072 and 2^17 + 2^18 - 1 = 393215
+func NewFakeShortChanIDFromInt(chanID uint64) ShortChannelID {
+	c := NewShortChanIDFromInt(chanID)
+	c.BlockHeight = (c.BlockHeight >> 6) + (1 << 17)
```

Comment:
What about the TxIndex and output? As otherwise, any zero conf channel accepted at the same block height may end up having the same scid?

Reply:
The chanID parameter is misleading here. This method actually gets a pending channel id (which is the random number that is generated for pending channels) and generate a new fake channel id from it. Since the pending is already a big random number we shouldn't have a collision in practice.
I will change the naming to be more clear.